### ISSUE TITLE
Problems getting started with FreeV2G 

### ISSUE THREAD

**Issue by lho-stx** [Sevenstax/FreeV2G#156]
Hi,

I am facing exactly same problem, so decided not to open a separate issue. If this is a problem, feel free to move/delete this post.

**Your Setup**
Platform: Whitebeet EVSE + Whitebeet EV
Firmware Version: i.e. V02_00_01 and V01_00_06 
Host Controller Interface: Both boards connected to simple L2 ethernet switch which is connected to PC
Host: FreeV2G EV_v1.0.6_1 + EVSE_v2.0.1_4

**Describe the bug**
We did not receive a frame before the timeout of 30s

**To Reproduce**
I started two instances of Application.py from the same PC. Until I connect the EVSE and EV board everything looks fine. One board is waiting for the EV and the other one is waiting for EVSE. Once I connect the GND, CP, PE the EVSE side reports "We did not receive a frame before the timeout of 30s" error.

**Expected behavior**
Expect a response to begin charging from vehicle, e.g. "Session started" received" printed to terminal.

**Logs**
At the moment I have outputs from Application.py and EVSE UART output.
[evse_response.txt](https://github.com/Sevenstax/FreeV2G/files/10803657/evse_response.txt)
[ev_response.txt](https://github.com/Sevenstax/FreeV2G/files/10803658/ev_response.txt)

**Additional context**
After we received the WB from Codico we updated the boards to the latest version. We didn't change the PIB config file.

_Originally posted by @crtg in https://github.com/Sevenstax/FreeV2G/issues/154#issuecomment-1440003683_

**Comment by Dalachowsky** [comment]
I have encountered the same problem today. I had to run FreeV2G for EV and EVSE on two separate PC's to make it work. It seems like some issue with receiving frames when two Whitebeets are sending data to the same eth interface.

**Comment by lho-stx** [comment]
It can lead to saturation of network traffic, when you are connecting two Whitebeets to a switch in order to use only one ethernet interface on your host, because you are routing a closed ring, when you connect the CP lines. Please try to use multiple ethernet interfaces on your PC (a cheap ETH-USB dongle will work) or two separeted PCs.

lho

**Comment by crtg** [comment]
Hi,

thanks for the tip. I have used two separate interfaces and the demo charging process works almost as expected. However, if restart the scripts after the charging (without cycling the power supply of the WB) and I leave the EVSE board waiting for the EV for more than 30s the Application.py will crash with the following output:

Wait until an EV connects
Traceback (most recent call last):
  File "/home/elektro/Desktop/FreeV2G-EVSE/Whitebeet.py", line 175, in _sendReceive
    raise Warning("Response from mod ID {:02X} with unexpected sub ID {:02X} received".format(response.mod_id, response.sub_id))
Warning: Response from mod ID 29 with unexpected sub ID 81 received

UART
[ 2514.172] [NOTICE] SLAC: stxSLAC_Stop()
[ 2514.184] [NOTICE] SLAC: stxSLAC_Start()
[ 2514.327] [NOTICE] V2GCPS: Change State from ERR to A
[ 2514.327] [NOTICE] V2GCPS: stxXBAR_V2GCPS_NotifyHandler()
[ 2514.328] [NOTICE] V2GCPS: # V2GCPS_INFOTYPE_CP_STATE
[ 2514.328] [NOTICE] V2GCPS: STATE A

This error doesn't appear the first time after the startup. If I restart the WB (power supply off/on) the session will be established. I have enabled the mirror setting and the output is attached. [evse_dump.pcapng.txt](https://github.com/Sevenstax/FreeV2G/files/10825616/evse_dump.pcapng.txt)


Can you please provide some hints?

Thanks and best regards, Črt

**Comment by lho-stx** [comment]
Hey @crtg,

if you did not connect your EV, this is intended. There is a built in timeout of 30 seconds. 
If you connected your EV and the application is failing I need some more time to look into this. At first glance I could not reproduce this problem. I kindly ask you to restart the application without disconnecting the CP lines. Does the application starts successful in this case?

Best regards,
lho

**Comment by crtg** [comment]
Hi,
thanks for your feedback. No, the EV was not connected. However, the EVSE module waits for EV indefinitely for the first time. It seems that timeout is only after the second session. Does the application exit normally or does it crash in case of timeout? I will replicate the problem and write you more details, but at the moment I am testing different configuration (EV board + DC charger).

Thanks for now and best regards, Črt

**Comment by lho-stx** [comment]
Hey @crtg,

I am currently looking into this. I have the same issue. The intended behavior is the first case: Whitebeet should wait for an EV or exit gracefully (e.g. when EV is connected but SLAC failed).

I need some more time to find the problem.

Thank you for your patience,
lho

**Comment by crtg** [comment]
Hi,

thanks for your help.

Regards, Črt
